### ISSUE TITLE
TDecimate wipes ALL frame properties when outputting hybrid (blended) frames

### ISSUE THREAD

**Issue by flossy83** [pinterf/TIVTC#40]
I can imagine a reason for it working this way - hybrid frames are blended and so their properties would be blended too, which is kind of an invalid concept.  

But I am writing my own custom frame properties into my clip before it arrives at TDecimate.  I need to pass these properties downstream but can't cause TDecimate deletes them.

It may be possible to do a bodge workaround of sidechaining a separate clip and merging its properties back in with the output of TDecimate.

**Comment by pinterf** [comment]
Yes, I can see cases in the code where blending of two frame occur, and the frame properties are not derived from either frame. I'm gonna fix it.

**Comment by flossy83** [comment]
Thank you, I can confirm it works as expected. 

I did notice the new TFM frame props are still getting deleted for hybrid blended frames - I presume this is intentional?

**Comment by pinterf** [comment]
I don't know it yet. Next week I go on with investigating.
Maybe it's clear but I'm not in the picture with this plugin right now; regarding "New TFM frame props", which ones do you mean?
